src/components/Counter.js

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
```

Comment:
By this way the code is importing everything from react even is not required all functionalities here. The code can be updated by this way.

`import React, {useState} from 'react';
`